docs-chef-io/content/inspec/resources/aws_api_gateway_domain_name.md

```diff
@@ -0,0 +1,191 @@
+
+: **Field**: `distribution_domain_name`
+
+`distribution_hosted_zone_id`
+: The region-agnostic Amazon Route 53 Hosted Zone ID of the edge-optimized endpoint. The valid value is Z2FDTNDATAQYW2 for all the regions.
+
+: **Field**: `distribution_hosted_zone_id`
+
+`endpoint_configuration_types`
+: The endpoint type. The valid values are EDGE for edge-optimized API setup, most suitable for mobile applications; REGIONAL for regional API endpoint setup, most suitable for calling from AWS Region; and PRIVATE for private APIs.
+
+: **Field**: `endpoint_configuration.types`
+
+`endpoint_configuration_vpc_endpoint_ids`
+: A list of VpcEndpointIds of an API (RestApi) against which to create Route53 ALIASes. It is only supported for PRIVATE endpoint type.
+
+: **Field**: `endpoint_configuration.vpc_endpoint_ids`
+
+`domain_name_status`
+: The status of the DomainName migration. The valid values are AVAILABLE and UPDATING . If the status is UPDATING , the domain cannot be modified further until the existing operation is complete. If it is AVAILABLE , the domain can be updated.
+
+: **Field**: `domain_name_status`
+
+`domain_name_status_message`
+: An optional text message containing detailed information about status of the DomainName migration.
+
+: **Field**: `domain_name_status_message`
+
+`security_policy`
+: The Transport Layer Security (TLS) version + cipher suite for this DomainName. The valid values are TLS_1_0 and TLS_1_2 .
+
+: **Field**: `security_policy`
+
+`tags`
+: The collection of tags.
+
+: **Field**: `tags`
+
+`mutual_tls_authentication.truststore_uri`
+: An Amazon S3 URL that specifies the truststore for mutual TLS authentication, for example s3://bucket-name/key-name . The truststore can contain certificates from public or private certificate authorities. To update the truststore, upload a new version to S3, and then update your custom domain name to use the new version. To update the truststore, you must have permissions to access the S3 object.
+
+: **Field**: `mutual_tls_authentication.truststore_uri`
+
+`mutual_tls_authentication.truststore_version`
+: The version of the S3 object that contains your truststore. To specify a version, you must have versioning enabled for the S3 bucket.
+
+: **Field**: `mutual_tls_authentication.truststore_version`
+
+`mutual_tls_authentication_truststore_warnings`
+: A list of warnings that API Gateway returns while processing your truststore. Invalid certificates produce warnings. Mutual TLS is still enabled, but some clients might not be able to access your API. To resolve warnings, upload a new truststore to S3, and then update you domain name to use the new version.
+
+: **Field**: `mutual_tls_authentication.truststore_warnings`
+
+`ownership_verification_certificate_arn`
+: The ARN of the public certificate issued by ACM to validate ownership of your custom domain. Only required when configuring mutual TLS and using an ACM imported or private CA certificate ARN as the regionalCertificateArn.
```

Comment:
@soumyo13 - do we need to include the **RegionalCertificateArn** field?

Reply:
and, is this a one word here: https://docs.aws.amazon.com/cloudfront/index.html

same in plural resource too!

Reply:
We can remove the second line from both the docs files.